README.md

```diff
@@ -32,6 +32,7 @@ log-based layout in files.
 * Current build status:
   [![MPICH](https://github.com/Parallel-NetCDF/E3SM-IO/actions/workflows/mpich_static.yml/badge.svg)](https://github.com/Parallel-NetCDF/E3SM-IO/actions/workflows/mpich_static.yml)
   [![OpenMPI](https://github.com/Parallel-NetCDF/E3SM-IO/actions/workflows/ubuntu_ompi.yml/badge.svg)](https://github.com/Parallel-NetCDF/E3SM-IO/actions/workflows/ubuntu_ompi.yml)
+* Run E3SM-IO with Cache VOL and Async VOL: see [cache_async_vol.md](./cache_async_vol.md)
```

Comment:
This bullet should goes to docs/INSTALL.md.
Also, move cache_async_vol.md to folder docs.